apps/browser-extension-wallet/src/components/TrezorDerivationTypeSelector.tsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { Select, Box, InfoComponent, Tooltip } from '@input-output-hk/lace-ui-toolkit';
+import { DerivationType } from '@lace/cardano';
+import styles from './TrezorDerivationTypeSelector.module.scss';
+
+interface TrezorDerivationTypeSelectorProps {
+  value?: DerivationType;
+  onChange: (derivationType: DerivationType) => void;
+  disabled?: boolean;
+}
+
+export const TrezorDerivationTypeSelector: React.FC<TrezorDerivationTypeSelectorProps> = ({
+  value,
+  onChange,
+  disabled = false
+}) => {
+  const { t } = useTranslation();
+
+  const derivationTypes = [
+    {
+      value: 'ICARUS',
+      label: t('core.derivationType.icarus', 'Icarus'),
+      description: t(
+        'core.derivationType.icarusDescription',
+        'Most common for wallets created with software applications'
```

Comment:
Having fallback strings is unnecessary
```suggestion
        'core.derivationType.icarusDescription'
```